pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -230,6 +248,21 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 		klog.InfoS("Static policy: Allocate", "pod", klog.KObj(pod), "containerName", container.Name)
 		// container belongs in an exclusively allocated pool
 
+		if p.options.SMTAlignmentEnabled && numCPUs%p.topology.CPUsPerCore() != 0 {
```

Comment:
Can we add some parentheses here to make this slightly more clear, i.e.
```
		if p.options.SMTAlignmentEnabled && ((numCPUs%p.topology.CPUsPerCore()) != 0) {

```

Reply:
ack